internal/pkg/admission/validating-webhook.go

```diff
@@ -73,32 +73,32 @@ func (r *Reconciler) validatingWebhookConfiguration(
 		Port:      &admissionPort,
 	}
 
-	sideEffects := clusterAdmissionPolicy.Spec.SideEffects
+	sideEffects := policy.GetSideEffects()
 	if sideEffects == nil {
 		noneSideEffects := admissionregistrationv1.SideEffectClassNone
 		sideEffects = &noneSideEffects
 	}
 	return &admissionregistrationv1.ValidatingWebhookConfiguration{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: clusterAdmissionPolicy.Name,
+			Name: policy.GetName(),
```

Comment:
If I'm understanding this correctly, if there are two `AdmissionPolicy` named in the same way in two different namespaces, this will clash when the webhook is reconciled (here and on the mutating one).

I think we need to also include the namespace, so if this happens, cluster wide resources (like webhook registrations) don't collide.

Reply:
you are right, good catch! This was also a problem with the configmap as two policies with the same name in different namespaces would have the same configmap entry.
I'm using namespace-policyname for the AdmissionPolicy name now. See changes in https://github.com/kubewarden/kubewarden-controller/pull/177/commits/d1ff8f8c71cf844e3f310d60e812f92f27f3dc3b